frontend/src/components/main-view/details/supplier-details/accordion/list-accordion/ListAccordion.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+//component imports
+
+import SupplierListItem from "./supplier-list-item/SupplierListItem";
+import {List,} from "@mui/material";
```

Comment:
```suggestion
import {List} from "@mui/material";
```
import {List,} from "@mui/material";